agent/api/containerstatus.go

```diff
@@ -151,6 +180,43 @@ func (cs *ContainerStatus) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + cs.String() + `"`), nil
 }
 
+// UnmarshalJSON overrides the logic for parsing the JSON-encoded container health data
+func (healthStatus *ContainerHealthStatus) UnmarshalJSON(b []byte) error {
+	if strings.ToLower(string(b)) == "null" {
+		*healthStatus = ContainerHealthUnknown
+		return nil
+	}
+
+	if b[0] != '"' || b[len(b)-1] != '"' {
+		*healthStatus = ContainerHealthUnknown
+		return errors.New("containerHealthStatus must be a string or null; Got " + string(b))
```

Comment:
Same here. Please reformat this as per go standard for error strings:
`"container health status: ..."`